jobs.py

```diff
@@ -341,6 +341,9 @@
 BROKEN_TESTS = [
     # not really broken but has complex prerequisites (3 NICs on 3 different networks)
     "tests/migration/test_host_evacuate.py::TestHostEvacuateWithNetwork",
+    # running quicktest on zfsvol generates dandling TAP devices that are hard to
```

Comment:
I think you mean "dangling".
```suggestion
    # running quicktest on zfsvol generates dangling TAP devices that are hard to
```

Reply:
dandling is more poetic but yes it is dangling like dangling pointers :)

Reply:
It's also in the commit message